tendrl/monitoring_integration/upgrades/delete_dashboards.py

```diff
@@ -51,6 +52,19 @@ def delete_dashboards(server_ip, user, password):
 
 def main():
     try:
+        print ("\n Migrating graphite data \n")
+        os.system(
+            "PYTHONPATH=$GRAPHITE_ROOT/webapp/ "
+            "django-admin migrate --fake dashboard "
+            "--settings=graphite.settings "
+            "--run-syncdb"
+        )
+        os.system(
```

Comment:
A fake command is only for the already existing table or app (dashboard), so after that, I am running the fake-initial command for all tables or apps to migrate data.  If I run fake-initial command first then it will throw an error for dashboard table.

In this way we can run this command any number of time, it won't give any error.  If migration is present it will migrate otherwise it will display no migration found.

Reply:
Ok.